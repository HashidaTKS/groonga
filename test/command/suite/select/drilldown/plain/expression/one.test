table_create Users TABLE_HASH_KEY ShortText
column_create Users first_name COLUMN_SCALAR ShortText
column_create Users last_name COLUMN_SCALAR ShortText
column_create Users nickname COLUMN_SCALAR ShortText

load --table Users
[
{"_key": "alice1", "nickname": "My Alice"},
{"_key": "alice2", "first_name": "My", "last_name": "Alice"},
{"_key": "bob", "nickname": "Bob"}
]

select Users \
  --drilldown "nickname || first_name + ' ' + last_name"
