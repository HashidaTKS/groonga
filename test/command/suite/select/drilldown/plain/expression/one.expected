table_create Users TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Users first_name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Users last_name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Users nickname COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Users
[
{"_key": "alice1", "nickname": "My Alice"},
{"_key": "alice2", "first_name": "My", "last_name": "Alice"},
{"_key": "bob", "nickname": "Bob"}
]
[[0,0.0,0.0],3]
select Users   --drilldown "nickname || first_name + ' ' + last_name"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        3
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "first_name",
          "ShortText"
        ],
        [
          "last_name",
          "ShortText"
        ],
        [
          "nickname",
          "ShortText"
        ]
      ],
      [
        1,
        "alice1",
        "",
        "",
        "My Alice"
      ],
      [
        2,
        "alice2",
        "My",
        "Alice",
        ""
      ],
      [
        3,
        "bob",
        "",
        "",
        "Bob"
      ]
    ],
    [
      [
        2
      ],
      [
        [
          "_key",
          "ShortText"
        ],
        [
          "_nsubrecs",
          "Int32"
        ]
      ],
      [
        "My Alice",
        2
      ],
      [
        "Bob",
        1
      ]
    ]
  ]
]
